tools/showcase/Makefile

```diff
@@ -5,7 +5,7 @@ setup:
 	python3 -m venv venv
 	./venv/bin/python -m pip install --upgrade pip
 	./venv/bin/python -m pip install -r requirements.txt
-	./venv/bin/python -m playwright install
+	./venv/bin/python -m playwright install-deps ## was formerly install only 
```

Comment:
Seems like this is only needed if you didn't have NodeJS 16+ already installed. We have https://github.com/h2oai/wave/blob/main/.nvmrc set properly. Can you try using a regular `./venv/bin/python -m playwright install` with a proper NodeJS version and see if it works for you?

```suggestion
	./venv/bin/python -m playwright install-deps
```

Reply:
Oh This
will rerun this in a bit and get back to you

Reply:
![Screenshot 2023-07-07 093013](https://github.com/h2oai/wave/assets/60529828/d5a8cce5-c21a-4677-aa40-2a9493ad0291)

I used the nvrms as instructed and changed my version to 16 but still the same behaviour

Reply:
What does "same behavior" mean? From the screenshot above it seems like you already have the browsers installed.

Can you try using the old Makefile (with `playwright install`)

```
cd tools/showcase
make clean
make setup
make generate
```
and see if it runs properly with Node 16 set?

Reply:
Same behaviour as in the first line does nothing but the second line initiates the process of installing
I will run the commands you just specified though

Reply:
> Same behaviour as in the first line does nothing

That's fine. It should install the necessary browsers. If you already have them installed, it should do nothing. The important part is if you are able to run `make generate` properly.

Reply:
So do I revert my change to the makefile then

Reply:
If the steps I outlined above work for you properly (images for docs are generated properly), then yes, revert your changes and let's go with the original Makefile.

Reply:
Done and committed already

Reply:
Alright, cleaning and running now

Reply:
Confirmed 
Using 16+ fixed it 
so good to go
my last commit reverted the makefile